### ISSUE TITLE
Results Map: visualization doesn't work well when we only deployed into a subset of a neighborhood

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2815]
The Results Map in Chicago (but could impact all cities) does not accurately portray our data. We only audited small radii in each deployment neighborhood but this isn't visible and our scoring model doesn't take this into account either, which makes the deployed neighborhoods have a "low sidewalk problems/mile" rating:

![image](https://user-images.githubusercontent.com/1621749/158425859-29834a42-df23-4e02-b265-95e7c4dea066.png)

**Comment by misaugstad** [comment]
Our scoring model actually does take this into account. The score is really just based on issues per meter. The only thing that doesn't work the same is that we only show a color if at least 30% of a neighborhood has been audited. Right now, it works out that only 30% of the streets that are available need to be audited.

So the scale for the score is still the same, the map just suggests that it represents the entire neighborhood instead of a small section of it.

@jonfroehlich given this, do you still want us to remove the Results Map for now? A la #2816

**Comment by jonfroehlich** [comment]
Hmm, interesting. In that case, maybe we boost this threshold to something more like 60-70%. Thoughts?

**Comment by misaugstad** [comment]
Yeah I'm fine with that. Note that this won't change anything in Chicago, since all of the available streets have been audited at least once. Another change I could make would be using the percentage of the available streets that have been audited by "high quality" users. That would actually make a lot more sense, since low quality data doesn't get incorporated into clustering, and the Results map uses those clusters instead of raw label counts.

**Comment by jonfroehlich** [comment]
Oh, I thought it was all streets in the neighborhood—but yah, makes sense.

Agree with your suggestion.

However, that still won't fix the problem with the fact that not all of the streets are allocated in those Chicago neighborhoods (because we did a finer deployment granularity). And so this 30% threshold cutoff (or whatever we make it), doesn't make sense . Because Results Map visualizes things at the neighborhood level.

**Comment by misaugstad** [comment]
Shall I remove this map for only Chicago then?

**Comment by jonfroehlich** [comment]
Well, the longer term fix is probably to make the cutoff threshold take into account the entire neighborhood (rather than just the currently allocated streets) or to show smaller assigned regions than neighborhoods (if smaller or different geographic granularities were used).

But the short-term fix is probably to remove for Chicago for now, yes. :)

**Comment by misaugstad** [comment]
Closing this as we're going to be retiring the Results page.
